presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/TypeConverter.java

```diff
@@ -87,6 +91,9 @@
 
 public final class TypeConverter
 {
+    public static final String ORC_ICEBERG_ID_KEY = "iceberg.id";
```

Comment:
The specification for this is out of date and says this is `iceberg.column.id` in the ORC user metadata. Can you file an issue to update the specification?

Reply:
Sure, I've commented on the PR which updated the spec.